packages/docusaurus/lib/client/App.js

```diff
@@ -18,7 +18,7 @@ function App() {
   const [context, setContext] = useState({});
   return (
     <DocusaurusContext.Provider
-      value={{siteConfig, ...metadata, ...context, setContext, routes}}>
```

Comment:
Not used in context. Should we include it ?

Reply:
What is not used?